var/spack/repos/builtin/packages/highfive/package.py

```diff
@@ -11,11 +11,11 @@ class Highfive(CMakePackage):
     """HighFive - Header only C++ HDF5 interface"""
 
     homepage = "https://github.com/BlueBrain/HighFive"
-    url      = "https://github.com/BlueBrain/HighFive/archive/v2.0.tar.gz"
+    url      = "https://github.com/BlueBrain/HighFive/archive/v2.1.tar.gz"
     giturl   = "https://github.com/BlueBrain/HighFive.git"
 
     version('develop', git=giturl)
-    # todo : waiting for author to release new version
+    version('2.1', 'cc9e93baecc939c6984f220643338092b7e71ef666cb1e1c80f3dfde0eaa89f2')
```

Comment:
@alkino why dont we use the tag here?

Reply:
I follow the rest of the file.

Reply:
I see... @pramodk any advantage/disadvantage of using release tarballs?

Reply:
did you find problems with parallel hdf5? last time I tested it was fine

Reply:
https://github.com/BlueBrain/HighFive/pull/242#issuecomment-548347869